docs/esa/hubble/hubble.rst

```diff
@@ -82,11 +83,11 @@ Calibration levels can be RAW, CALIBRATED, PRODUCT or AUXILIARY.
 
 Note: Artifact is a single Hubble product file.
 
-.. code-block:: python
+.. doctest-skip::
```

Comment:
why skip? add comment or run the code

Reply:
It doesn't run, I've marked it as such. @jespinosaar can you replace this with one that does run, or is the file in question just an example?